lib/haml_coffee_assets/rails/engine.rb

```diff
@@ -38,6 +38,9 @@ class Engine < ::Rails::Engine
             env.register_mime_type 'text/hamlc', extensions: ['.hamlc', '.jst.hamlc']
             env.register_transformer 'text/hamlc', 'application/javascript', ::HamlCoffeeAssets::Transformer
             env.register_transformer 'text/hamlc', 'application/javascript+function', ::HamlCoffeeAssets::Transformer
+            # support for chaining via ERB, documented via https://github.com/rails/sprockets/pull/807
+            env.register_mime_type 'text/hamlc+ruby', extensions: ['.hamlc.erb', '.jst.hamlc.erb']
+            env.register_transformer 'text/hamlc+ruby', 'text/hamlc', ::Sprockets::ERBProcessor
```

Comment:
This isn't exactly the same as #190 but equivalent right?

Reply:
They are different, @mathieujobin. #190 fixes hamlc without .erb